wordlist

```diff
@@ -635,7 +635,7 @@ race:france
 race:italy
 race:usa
 radix
-rc
+rc[0-9]*
```

Comment:
should it be a `+`?
```suggestion
rc[0-9]+
```

Reply:
"rc" without numbers was already in the wordlist, so it may be good to allow zero digits here to keep "rc" a valid word.

Reply:
Yeah, * was to cover both `rc` and `rc1`.